### DISCUSSION TITLE
dar-backup version 1.4.3 released

### DISCUSSION THREAD

**Discussion by per2jensen** [per2jensen/dar-backup#23] [Announcements]
This is a maintenance/bugfix release with some changes, no new functionality.

During the yearly backups, I encountered a bug when generating the par2 files. It occurred only on the file system mounted from the cloud. I ended up refactoring how the par2 files are generated, which had the added benefit of a more simple setup.

- par2 now generated from within the script - not using the dar integrated system (allows for async par2 generation on the remote server)
- a small fix in the installer to make sure scripts are executable
- no exit if a backup definition fails, do the next one

SHA256:
898212dad3a3dfd4dd1f105acdd67d0401c7038f3808b76788e3f40f853a4459  dar-backup-linux-v1.4.3.tar.gz
SUCCESS: a release tarball from tag: "v1.4.3" was produced

<hr /><em>This discussion was created from the release <a href='https://github.com/per2jensen/dar-backup/releases/tag/v1.4.3'>dar-backup version 1.4.3 released</a>.</em>

**Comment by klamanch** [comment]
Thank you for sharing this great work. I only heard about DAR recently. This is really nice in how you have incorporated everything.
Best Regards!

**Reply by per2jensen** [reply]
Thank you for your kind comment @klamanch  - I am glad, if the scripts are of use to you.
